### PR TITLE
movies list

### PR DESCRIPTION
**Pull request by stunnerboyone**
[DEMO LINK](https://stunnerboyone.github.io/react_movies-list-add-form/)

### CODE REVIEW

File: src/components/NewMovie/NewMovie.tsx [hunk 1]
```diff
@@ -1,45 +1,93 @@
...
+  onAdd: (newMovie: Movie) => void;
+};
+
+export const NewMovie = ({ onAdd }: Props) => {
+  const [count, setCount] = useState(0);
+  const [title, setTitle] = useState('');
+  const [description, setDescription] = useState('');
+  const [imgUrl, setImgUrl] = useState('');
+  const [imdbUrl, setImdbUrl] = useState('');
+  const [imdbId, setImdbId] = useState('');
```
Comment (etojeDenys) [inline src/components/NewMovie/NewMovie.tsx:15]: we can combine it in one useState
```
const [newMovie, setNewMovie] = useState({
  title: '',
  ...
})
```

now we can also create only one handle function for all fields
```
const handleChange = (e: ...) => {
  const {name, value} = e.target;
  setMovie((prevMovie) => ({...prevMovie, [name]: value}))
}
```

don't forget to specify name attr to input in TextField

**Review by etojeDenys — CHANGES_REQUESTED** [review]
let's show the error when I try to fill inputs with spaces

File: src/components/NewMovie/NewMovie.tsx [hunk 2]
```diff
@@ -1,45 +1,93 @@
...
+
+    setCount(count + 1);
+
+    setNewMovie({
+      title: '',
+      description: '',
+      imgUrl: '',
+      imdbUrl: '',
+      imdbId: '',
+    });
```
Comment (anastasiia-tilikina) [inline src/components/NewMovie/NewMovie.tsx:43]: let's create a var for example `initialState` and reuse it to avoid code duplication

**Review by anastasiia-tilikina — CHANGES_REQUESTED** [review]
just one more small fix

**Review by anastasiia-tilikina — APPROVED** [review]
Well done!

**Pull request closed without merging**